src/app/lib/__snapshots__/regex.const.test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`regex.const AMP_REGEX should match snapshot 1`] = `/\\\\\\.amp\\$/`;
+
+exports[`regex.const TLD_REGEX should match snapshot 1`] = `/\\(\\\\\\.com\\|\\\\\\.co\\\\\\.uk\\)/g`;
+
+exports[`regex.const default should match snapshot 1`] = `
+Object {
+  "AMP_REGEX": /\\\\\\.amp\\$/,
+  "TLD_REGEX": /\\(\\\\\\.com\\|\\\\\\.co\\\\\\.uk\\)/g,
```

Comment:
I'm not really sure what benefit snapshot tests have for consts? If someone were to change them, there would just be two changes instead of one in a PR, just trying to think how the protect us from a regression 🤔

Reply:
Yeah, this is probably overkill, I'll remove it.